functest_requirements.txt

```diff
@@ -1,3 +1,4 @@
+ansible-core==2.13.8
```

Comment:
what if we say `ansible-core!=2.13.9`?

Reply:
It's also broken in 2.14.5 which will come in with new versions of python, so !=2.13.9 isn't technically correct.

Reply:
So we can still explicitely reject all broken versions:
`ansible-core!=2.13.9,!=2.14.5`